octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
```

Comment:
[Severity: Minor] The package declaration is present, but it is good practice to include a comment at the top of the file describing the purpose of the class.
Consider adding a brief comment explaining the purpose of the Calculator class.
Rationale: This helps other developers understand the intent of the class quickly and improves maintainability.
Reference: Java Code Conventions.